src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,18 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null
+                && userService.findByEmail(email).getPassword().equals(password)) {
```

Comment:
let's create a variable for the result of findByEmail, so we won't need to call it twice